examples/client.rs

```
@@ -47,10 +47,9 @@ async fn run_client() -> io::Result<()> {
     let tls = match ca {
         Some(ref mut rd) => {
             // Read trust roots
-            let certs = rustls_pemfile::certs(rd)
-                .map_err(|_| error("failed to load custom CA store".into()))?;
+            let certs = rustls_pemfile::certs(rd).flat_map(|x| x);
```

I don't think we should ignore errors here.